### PR TITLE
Return of "OH SHIT, ANTS!!!!!!"

### PR DESCRIPTION
**Pull request by Carton171**
## About the PR 
Returns "OH SHIT, ANTS!!!!" to its rightful place.
Makes ants cause you to scream again, with an extended cooldown to mitigate audible spam.

## Why's this needed? 
The "OH SHIT, ANTS!!!!!" message is both hilarious and a part of goonstation heritage, and it's a shame to lose it. 

## Changelog 

```changelog
(u)Carton171
(+) Players will once again lose their shit upon being filled with/covered in ants
```

### CODE REVIEW

**Comment by DecidedlyUnethical** [comment]
Soul

File: code/modules/chemistry/Reagents-Misc.dm [hunk 1]
```diff
@@ -1706,7 +1706,14 @@ datum
...
 				if (method == TOUCH)
 					. = 0 // for depleting fluid pools
+				if (!ON_COOLDOWN(M, "ants_scream", 15 SECONDS)) //lets make it less spammy
+					M.emote("scream")
+					if (method == INGEST || method == INJECT)
+						boutput(M, SPAN_ALERT("<b>OH SHIT, ANTS [pick("", "IN MY BLOOD", " IN MY VEINS")]![pick("", "!", "!!", "!!!", "!!!!")]</b>"))
+					else
+						boutput(M, SPAN_ALERT("<b>OH SHIT, ANTS![pick("", "!", "!!", "!!!", "!!!!")]</b>"))
 				random_brute_damage(M, 4)
+				return
```
Comment (frawhst) [inline code/modules/chemistry/Reagents-Misc.dm:1716]:
```suggestion
```
nah you don't need this at the end. every proc basically `return`s by default at the end

File: code/modules/chemistry/Reagents-Misc.dm [hunk 2]
```diff
@@ -1706,7 +1706,14 @@ datum
 					return
 				if (method == TOUCH)
 					. = 0 // for depleting fluid pools
+				if (!ON_COOLDOWN(M, "ants_scream", 15 SECONDS)) //lets make it less spammy
```
Comment (frawhst) [inline code/modules/chemistry/Reagents-Misc.dm:1709]: Also I authored the original scream spam reduction change, but 15 seconds is HARSH. Drop to 10 imo, or even 5.

**Review by TobleroneSwordfish — APPROVED** [review]
🐜

**Comment by MeggalBozale** [comment]
Im late to this but soul :heart:

**Pull request merged**